xcode/Subconscious/Shared/Config.swift

```diff
@@ -56,7 +56,7 @@ struct NoosphereConfig: Equatable, Codable {
     /// NOTE: In future, we might support multiple spheres. If so, this
     /// flag will be deprecated in favor of multiple spheres
     var sphereStoragePath = "sphere"
-    /// Default owner key name for spheres on this device
-    var ownerKeyName = "User"
+    /// Owner key name for key that owns default sphere on this device.
+    var ownerKeyName = "Subconscious"
```

Comment:
I imagine for Touch ID, we may end up seeing sheets that say something like "Such-and-such wants to access [owner key name]".

Given we do talk about "your Subconscious", I think calling the default sphere owner key "Subconscious" makes a lot of sense. "Such-and-such wants to access Subconscious"